packages/connector-aliyun-sms/src/types.ts

```diff
@@ -47,9 +47,6 @@ const templateGuard = z.object({
   type: z.nativeEnum(SmsTemplateType).default(2),
   usageType: z.string(),
   code: z.string(),
```

Comment:
will it be better to use `templateCode` here to avoid confusion between the dynamic codes? cc @wangsijie